iree/test/e2e/regression/generate_e2e_matmul_tests.py

```diff
@@ -144,6 +172,8 @@ def get_test_shapes(shapes_id: ShapesId):
         # running on fewer backends/drivers or with fewer generators
         # (see get_test_generators).
     ]
+  if shapes_id == ShapesId.GPU_LARGE:
+    return [TestShape(m=512, k=512, n=512)]
```

Comment:
Are you sure that you want to test only one shape where `M==K==N`?  I can understand that you want to test only one shape, and that you want it fairly large like this, but wouldn't you obtain a little more interesting test coverage by exercising `M`, `K`, `N` being distinct values, e.g. in case some compiler code inadvertently mixed up dims? @ThomasRaoux ?

Reply:
I agree with Benoit, it is fine to start with a single shape but picking `M==K==N` reduces the coverage area for no good reason.